habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -435,21 +324,21 @@ def load_checkpoint(self, checkpoint_path: str, *args, **kwargs) -> Dict:
     def _compute_actions_and_step_envs(self, buffer_index: int = 0):
         num_envs = self.envs.num_envs
         env_slice = slice(
-            int(buffer_index * num_envs / self._nbuffers),
-            int((buffer_index + 1) * num_envs / self._nbuffers),
+            int(buffer_index * num_envs / self._agent.nbuffers),
+            int((buffer_index + 1) * num_envs / self._agent.nbuffers),
         )
 
         t_sample_action = time.time()
 
-        # sample actions
+        # Sample actions
         with inference_mode():
-            step_batch = self.rollouts.buffers[
-                self.rollouts.current_rollout_step_idxs[buffer_index],
-                env_slice,
-            ]
+            # TODO: Refactor this in the next multi-agent PR to maintain the current observation in the trainer rather than indexing into rollouts.
```

Comment:
```suggestion
            # TODO Andrew: Refactor this in the next multi-agent PR to maintain the current observation in the trainer rather than indexing into rollouts.
```

Reply:
Oops, I actually already fixed this.